### PR TITLE
add supply job to assignment

### PR DESCRIPTION
**Pull request by s3341458**

### CODE REVIEW

File: python/jobs.py [hunk 1]
```diff
@@ -446,6 +446,7 @@ class Assignment(sdk.python.entities.Entity):
     production_deadline = Property(datetime.datetime)
     assignment_deadline = Property(datetime.datetime)
     job = Property(Job, backref="assignments")
+    supply_job = Property(Job, backref="supply_assignment")
```
Comment (DanielCollins) [inline python/jobs.py]: I feel like you meant to put this on Assignment model.

A job has a job named job, and also a job has a job named assignment? Doesn't seem right.

Comment (Amtron) [reply python/jobs.py]: Yeah I think @DanielCollins is right.

Comment (Amtron) [reply python/jobs.py]: On second look it is on the Assignment model. The Assignment model is defined at the bottom of the jobs.py file; maybe it's time to move Assignments into it's own file?

Comment (DanielCollins) [reply python/jobs.py]: that seems like a good idea

Comment (DanielCollins) [reply python/jobs.py]: https://github.com/merchisdk/sdk/issues/145

**Pull request merged**